crossing.gemspec

```diff
@@ -29,7 +29,7 @@ DESC
   s.add_development_dependency('rubygems-tasks')
   s.add_development_dependency('simplecov')
 
-  s.add_runtime_dependency('aws-sdk', '>= 2.0.0')
-  s.add_runtime_dependency('trollop', '~ 2')
+  s.add_runtime_dependency('aws-sdk-s3')
```

Comment:
don't we need the kms gem as well?

Reply:
S3 pulls it in automatically according to `bundle install`

Reply:
Per Jesse, kms is pulled in transitively